spec/features/06_user_adds_reviews_for_shoe_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
```

Comment:
As a small picky note, if you have a commented out test, just exclude it from your PR.